packages/enzyme-adapter-react-16.3/src/ReactSixteenThreeAdapter.js

```diff
@@ -78,6 +78,33 @@ function flatten(arr) {
   return result;
 }
 
+function nodeTypeFromType(type) {
+  if (type && type === Portal) {
```

Comment:
if type is falsy, it won't === Portal, so i think this can be `type === Portal ? 'portal' : utilNodeTypeFromType(type)`